services/src/unit-test/java/uk/gov/moj/sdt/services/SubmitQueryServiceTest.java

```diff
@@ -106,19 +111,23 @@ public void setUp() {
         // Instantiate all the mocked objects and set them in the target
 
         mockConsumerGateway = EasyMock.createMock(IConsumerGateway.class);
+        mockCmcConsumerGateway = EasyMock.createMock(IConsumerGateway.class);
         mockGlobalParamCache = EasyMock.createMock(ICacheable.class);
         mockErrorMsgCacheable = EasyMock.createMock(ICacheable.class);
         mockBulkCustomerDao = EasyMock.createMock(IBulkCustomerDao.class);
+        responsesSummaryUtil = EasyMock.createMock(ResponsesSummaryUtil.class);
 
         final GenericXmlParser genericParser = new GenericXmlParser();
         genericParser.setEnclosingTag("targetAppDetail");
 
         submitQueryService = new SubmitQueryService(mockConsumerGateway,
+                                                    mockCmcConsumerGateway,
```

Comment:
add test cases related to mockCmcConsumerGateway

Reply:
Done